lib/pdf_utilities/pdf_validator.rb

```diff
@@ -40,7 +44,14 @@ class Validator
       # 'file' can be a File, Tempfile, or a String file path
       def initialize(file, options = {})
         @file = file
-        @options = DEFAULT_OPTIONS.merge(options)
+
+        @options = if Flipper.enabled?(:vba_documents_larger_page_size_limit)
+                     DEFAULT_OPTIONS.merge(
+                       { width_limit_in_inches: MAX_PAGE_WIDTH, height_limit_in_inches: MAX_PAGE_HEIGHT }
+                     ).merge(options)
+                   else
+                     DEFAULT_OPTIONS.merge(options)
+                   end
```

Comment:
Since this `PDFValidator` lives in the `lib` directory rather than the `vba_documents` module, what do you think about moving this `VBADocuments`-specific logic instead to the `VBADocuments` classes that call this utility class? I worry that having the logic here might discourage someone from reusing this class out of fear that they don't understand everything that it's doing.